_includes/functions/details-list.md

```diff
@@ -0,0 +1,15 @@
+<div class="{{include.class | default: 'qa_details'}} {{jekyll.environment}}" markdown="1">
+{% for i in (0..9) %}
```

Comment:
9 questions/answers seems a reasonable limit; should it be made explicit somewhere?